pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTestSet.java

```diff
@@ -788,43 +760,66 @@ public void testReload(boolean includeOfflineTable)
     JsonNode resultTable = queryResponse.get("resultTable");
     assertEquals(resultTable.get("dataSchema").get("columnNames").size(), schema.size());
     assertEquals(resultTable.get("rows").size(), 10);
+    assertEquals(queryResponse.get("totalDocs").asLong(), numTotalDocs);
+
     // Test aggregation query to include querying all segemnts (including realtime)
     String aggregationQuery = "SELECT SUMMV(NewIntMVDimension) FROM " + rawTableName;
     queryResponse = postQuery(aggregationQuery);
     assertEquals(queryResponse.get("exceptions").size(), 0);
 
-    // Test filter on all new added columns
-    String countStarQuery = "SELECT COUNT(*) FROM " + rawTableName
-        + " WHERE NewIntSVDimension < 0 AND NewLongSVDimension < 0 AND NewFloatSVDimension < 0 AND "
-        + "NewDoubleSVDimension < 0 AND NewStringSVDimension = 'null' AND NewIntMVDimension < 0 AND "
-        + "NewLongMVDimension < 0 AND NewFloatMVDimension < 0 AND NewDoubleMVDimension < 0 AND "
-        + "NewStringMVDimension = 'null' AND NewIntMetric = 0 AND NewLongMetric = 0 AND NewFloatMetric = 0 "
-        + "AND NewDoubleMetric = 0 AND NewBytesMetric = ''";
-    queryResponse = postQuery(countStarQuery);
-    assertEquals(queryResponse.get("exceptions").size(), 0);
-    assertEquals(queryResponse.get("resultTable").get("rows").get(0).get(0).asLong(), countStarResult);
+    //tests that reload is not needed on the table after reloading all segments from controller api
```

Comment:
(minor) Add a space and capitalize
```suggestion
    // Tests that reload is not needed on the table after reloading all segments from controller api
```